styles/index.scss

```diff
@@ -0,0 +1,51 @@
+:root {
+  font-family: 'Bitter', serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+//50 85 25
+.game-wrapper {
```

Comment:
I think we don't need to connect all the classes with the .game-wrapper, it compiles to: ".game-wrapper .header-content". I think they  can work as single classes without relation to the wrapper, then you could use the scss in a simpler way (less lines of code):
`
  .game {
    // costam
    &-wrapper {
      // costam
    }
    &-image {
      // costam
    }
  }
  .footer {
    // costam
  }
  .header {
    // costam
  }`

Reply:
Still, I don't know what are you talking about, we should have a call probably :)